packages/manager-wiki/stories/manager-ui-kit/components/Drawer/base-component-doc/drawer.stories.tsx

```diff
@@ -103,7 +103,7 @@ import { useState } from 'react';`,
   render: ({}) => {
     const [isOpen, setIsOpen] = useState(false);
 
-    function onOpenChange({ open }: DrawerOpenChangeDetail) {
+    function handleOpenChange({ open }: DrawerOpenChangeDetail) {
```

Comment:
You cannot directly modify code from a library owned by Manager Core Team.

I think this change is not mandatory so we could keep it like this :)